pkg/v1/remote/error_roundtrip_test.go

```diff
@@ -0,0 +1,54 @@
+package remote_test
```

Comment:
missing copyright header

Reply:
I think we need a presubmit for it - not all files have it.

Reply:
Also added